lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/FeatureService.java

```diff
@@ -204,7 +204,7 @@ private List<Feature> getInstalledFeaturesList(String documentURI) {
         return libertyWorkspace.getInstalledFeatureList();
       }
 
-      Path featureListJAR = LibertyUtils.findFileInWorkspace(documentURI, "ws-featurelist.jar");
+      Path featureListJAR = LibertyUtils.findFileInWorkspace(documentURI, "bin/tools/ws-featurelist.jar");
```

Comment:
Should change to use `Paths.get()` here also.

Reply:
Fixed.